src/lang/en.json

```
@@ -884,4 +886,4 @@
     "GrafanaOncallUrl": "Grafana Oncall URL",
     "Browser Screenshot": "Browser Screenshot",
     "What is a Remote Browser?": "What is a Remote Browser?"
-}
+}
```

change seems accidental

```suggestion
}

```